docs/use/commands/pmm-admin.md

```diff
@@ -380,7 +380,7 @@ FLAGS:
 ###### Enable all collectors
 
 To enable all collectors, pass the parameter `--enable-all-collectors` in the `pmm-admin add mongodb` command.
-This will enable `collstats`, `dbstats`, `indexstats`, and `topmetrics` collectors.
+This will enable `collstats`, `dbstats`, `indexstats`, `CurrentOp` and `topmetrics` collectors.
```

Comment:
```suggestion
This will enable `collstats`, `dbstats`, `indexstats`, `currentop` and `topmetrics` collectors.
```
Other collectors are written in lower case, let's use the same format